packages/p2p/src/components/app.scss

```diff
@@ -56,6 +56,28 @@
             padding: 3.5rem 2.5rem 0;
         }
     }
+
+    .dc-hint-box {
+        margin-bottom: 1rem;
+
+        &__icon {
+            @include mobile {
+                align-self: flex-start;
+                min-width: 16px;
+                min-height: 16px;
+            }
+        }
+
+        &--info {
+            background-color: rgba(255, 173, 58, 0.16);
+        }
+
+        @include mobile {
+            margin-left: 5vw;
+            margin-right: 5vw;
+            width: 90vw;
+        }
+    }
```

Comment:
Can we add this to the component rather than P2P only?

Reply:
This is done.